src/librustdoc/html/render/print_item.rs

```diff
@@ -1675,6 +1681,16 @@ where
     w.write_str("</code></pre>");
 }
 
+fn wrap_item_write<W, F>(w: &mut W, f: F)
```

Comment:
`Buffer` actually already implements the `fmt::Write` trait. Wouldn't it be simpler to update `wrap_item` instead of creating a new function?

Reply:
Yeah that makes perfect sense, update the function : )